native/c/zshmem.hpp

```diff
@@ -58,6 +58,10 @@ struct ZShmContext
   size_t size;
 };
 
+#ifdef SWIG
+extern "C"
+{
+#endif
```

Comment:
This technically can be removed after the PR below is merged:
- https://github.com/zowe/zowe-native-proto/pull/518

Reply:
resolved in ddd89ff95ee5319d471940ebd1fbeb857cca8bdd